### ISSUE TITLE
dim-web does not compile with recent node version

### ISSUE THREAD

**Issue by TheMeier** [1and1/dim#96]
WARN notsup Unsupported engine for karma@1.7.1: wanted: {"node":"0.10 || 0.12 || 4 || 5 || 6 || 7 || 8"} (current: {"node":"14.17.0","npm":"6.14.13"})

**Comment by miesi** [comment]
True, this well aged npm needs a rewrite in a language which has a clear commitment on stable APIs

**Comment by miesi** [comment]
We need to write another Web GUI, layer3domain aware this time, in a not so fast changing language.
